inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -112,6 +113,27 @@ def get_flow_plane(self,
 
         return cutting_plane_mesh
 
+    def get_force_coefficients(self):
+        """Get the force coefficients of the object in the WindTunnel."""
+
+        num_header_lines = 9
+        force_coefficients = pd.DataFrame(
+            columns=["time_step", "Cm", "Cd", "Cl", "Cl(f)", "Cl(r)"])
+
+        force_coefficients_path = os.path.join(self.sim_output_path,
+                                               "postProcessing", "forceCoeffs1",
+                                               "0", "forceCoeffs.dat")
+        force_coefficients_df = pd.read_csv(force_coefficients_path,
+                                            header=None)
+
+        # Skip the first lines of the header to read the force coefficients
+        # on the respective time_step.
+        force_coefficients.loc[0] = force_coefficients_df.loc[self.time_step +
+                                                              num_header_lines,
+                                                              0].split()
```

Comment:
It seems to be that what you're trying to achieve here can be done with a better manipulation of `read_csv`.

- Why are you defining headers by hand? Are they written in the file? They can most likely be read from there.
- The last line is cryptic. What are you doing there?

Reply:
I agree with Fábio. Maybe share an example of the dataframe to see if there is a better way of reading this!

Reply:
I added a clarification on the docstring and process the file differently. 

Basically, I am reading a .dat file with the force coefficients over various iterations. On the file header there are several lines that are useless, so I remove them. The goal is to provide the users with the labels and the force coefficients at a respective time step.